<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>292</width>
    <height>343</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="cal_selected_lb">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>210</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Selected Date :</string>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="cal_widget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>272</width>
     <height>183</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="cal_next_month">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Next Month</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cal_onoff_grid">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Grid On/Off</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Current Page :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cal_today">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Today</string>
   </property>
  </widget>
  <widget class="QLabel" name="cal_cur_page_lb">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>240</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="cal_prev_month">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Prev.Month</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cal_set_range">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>310</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Set Range</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cal_set_day">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>310</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>21-10-10</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
